docs/book/content/citations.md

```diff
@@ -1,4 +0,0 @@
-# Citations and use cases of OG-USA
```

Comment:
@jdebacker . Why did this go away?

Reply:
This file was removed because the citations file we use is in the main directory (`OG-USA/citations.md`).